resources/META-INF/plugin.xml

```diff
@@ -15,7 +15,7 @@
   <version>1.10.1</version>
   <vendor>Twitter, Inc.</vendor>
 
-  <idea-version since-build="182.0" until-build="182.*"/>
+  <idea-version since-build="182.0" until-build="183.*"/>
```

Comment:
If it is not backwards compatible with 182.*, we may have to do the following

```suggestion
  <idea-version since-build="183.0" until-build="183.*"/>
```

Reply:
I think we will need to change this since some APIs changed.